server/src/test/java/io/crate/integrationtests/InsertIntoIntegrationTest.java

```diff
@@ -1651,4 +1651,19 @@ public void test_insert_into_table_with_primary_key_with_default_clause_sets_rig
             "1\n"
         ));
     }
+
+    @Test
+    public void test_insert_from_union_with_null_literal() {
```

Comment:
does this test cover any special branch not covered by the analyzer test? I wonder if it is really needed.

Reply:
It triggered an assertion in the union operator and covers the changes in it, but I can turn it into a planner test

Reply:
ah right. planner test would be great if no big thing.